Function-like procedural macros in expression position as seen in 1.30 announcement are not stable yet

Only macros in statement position are stable in 1.30.

```console
error[E0658]: procedural macros cannot be expanded to expressions (see issue #38356)                                           
 --> src/main.rs:5:15                                                                                                          
  |                                                                                                                            
5 |     let sql = sql!(SELECT * FROM posts WHERE id=1);                                                                        
  |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

The stabilization hasn't landed in nightly yet either.

```console
error[E0658]: procedural macros cannot be expanded to expressions (see issue #54727)                                           
 --> src/main.rs:5:15                                                                                                          
  |                                                                                                                            
5 |     let sql = sql!(SELECT * FROM posts WHERE id=1);                                                                        
  |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                         
  |                                                                                                                            
  = help: add #![feature(proc_macro_hygiene)] to the crate attributes to enable
```

Mentioning @steveklabnik.

--- a/posts/2018-10-25-Rust-1.30.0.md
+++ b/posts/2018-10-25-Rust-1.30.0.md
@@ -73,18 +73,30 @@ thing the attribute is attached to, in this case, `fn index() {}` and the rest
 of the function's body.
 
 Function-like macros define macros that look like function calls. For
-example, an `sql!` macro:
+example, the [`gnome-class` crate](https://gitlab.gnome.org/federico/gnome-class)
+has a procedural macro that defines `GObject` classes in Rust:
 
 ```rust
-let sql = sql!(SELECT * FROM posts WHERE id=1);
+gobject_gen!(
+    class MyClass: GObject {
+        foo: Cell<i32>,
+        bar: RefCell<String>,
+    }
+
+    impl MyClass {
+        virtual fn my_virtual_method(&self, x: i32) {
+            // ... do something with x ...
+        }
+    }
+)
 ```
 
-This macro would parse the SQL statement inside of it and check that it's
-syntactically correct. This macro would be defined like this:
+This looks like a function that is taking a bunch of code as an argument.
+This macro would be defined like this:
 
 ```
 #[proc_macro]
-pub fn sql(input: TokenStream) -> TokenStream {
+pub fn gobject_gen(input: TokenStream) -> TokenStream {
 ```
 
 This is similar to the derive macro's signature: we get the tokens that

